pkg/webhooks/validation/controller/controller.go

```diff
@@ -196,15 +197,21 @@ func newController(
 		client: client,
 		queue:  workqueue.NewRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(1*time.Second, 5*time.Minute)),
 	}
-
+	webhookConfigName := util.GetWebhookConfigName(features.ValidationWebhookConfigName, o.Revision, o.WatchedNamespace)
 	webhookInformer := cache.NewSharedIndexInformer(
 		&cache.ListWatch{
 			ListFunc: func(opts metav1.ListOptions) (runtime.Object, error) {
 				opts.LabelSelector = fmt.Sprintf("%s=%s", label.IoIstioRev.Name, o.Revision)
+				if features.EnableEnhancedResourceScoping {
+					opts.FieldSelector = fmt.Sprintf("metadata.name=%s", webhookConfigName)
```

Comment:
in theory, would this work for everyone? Not just under EnableEnhancedResourceScoping?
I don't mind having it under EnableEnhancedResourceScoping either, just trying to fully understand